src/pages/achievement/control/subcomponents/controlPanelTools/controlPanelUtils/PrerequisiteAdder.tsx

```diff
@@ -0,0 +1,68 @@
+import { Button } from '@blueprintjs/core';
+import React, { useState } from 'react';
+
+import { AchievementItem } from '../../../../../../features/achievement/AchievementTypes';
```

Comment:
may possibly shorten these urls by using absolute import. (not sure if we are changing Coding Standards for these)

Reply:
Yes, we should take a look at handling the use of relative path and absolute path soon. 

Will need to discuss with everyone with regards to this, if not coding standards may become really confusing, especially for the incoming CP3108 Team. 

Tagging @jetkan-yk and @anthony-halim in this conversation

Reply:
Created an issue https://github.com/source-academy/cadet-frontend/issues/1376